### ISSUE TITLE
Quest: Translating the Journal

### ISSUE THREAD

**Issue by underlord-oberon** [plantain-gnocchis/wow-voiceover-tbc-wotlk#60]
Add-on version: AI_VoiceOver_v1.4.5
Data version: AI_VoiceOverData_VanillaExtra_v1.4

Uldaman / Hall of the Keepers / NPC: Remains of a Paladin / Quest: Translating the Journal
Link: https://www.wowhead.com/wotlk/quest=2318/translating-the-journal

Description: Despite the voice-over is perfect, the text is a description of actions. The narrator voice is more suitable in this case, instead of the male undead voice.

Completion: playback ok.

**Comment by carevass** [comment]
This should be fixed with new audio in AI_VoiceOverData_VanillaExtra_v1.5.
